app/src/main/java/com/example/demo/MainActivity.kt

```diff
@@ -35,6 +35,11 @@ fun Greeting(
     name: String,
     modifier: Modifier = Modifier,
 ) {
+
```

Comment:
🚫 **[ktlint]** <standard:no-empty-first-line-in-method-block> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>First line in a method block should not be empty